src/components/Header/Header.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import { uuid } from 'uuidv4'
+
+Header.propTypes = {
+  children: PropTypes.node,
+  uuid: PropTypes.string,
+}
+
+export default function Header({ children, uuid }) {
```

Comment:
uuid is not a prop that gets passed in from outside and I think you don't need it (as I said before)